scripts/boot_bitxhub.sh

```diff
@@ -0,0 +1,39 @@
+CURRENT_PATH=$(pwd)
+BUILD_PATH=${CURRENT_PATH}/build
+N=$1
+
+function splitWindow() {
+  tmux splitw -v -p 50
+  tmux splitw -h -p 50
+  tmux selectp -t 0
+  tmux splitw -h -p 50
+}
+
+function start() {
+  cd ${CURRENT_PATH}
+  rm -rf build
+  tar -xf build.tar.gz
```

Comment:
rename build.tar.gz to bitxhub-version.tar.gz

Reply:
pack this with dynamic version and commit hash now.